### ISSUE TITLE
Range Finder WA does not work for Arcane spec that has 1 point in range.

### ISSUE THREAD

**Issue by ne7ex** [WeakAuras/WeakAuras2#4230]
### Description

Hey guys, after the recent update my range finder for the mage stopped working. 
It works perfectly fine when im in fire spec with 2 points in range talents, and it works for the arcane spec if im not specced into any range talents. Also works on my warlock . 

Its just stuck at 33-35 no matter where I stand from the target.

Here's a video example of what's happening : https://streamable.com/db7ty1

### WeakAuras Version

5.3.4

### World of Warcraft Flavor

Wrath of the Lich King Classic

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Reproduction Steps

Don't know how to replicate something that already isn't working by default in the arcane tree with 1 specced into "Magic Attunement". 

### Last Good Version

5.3.3

### Screenshots

![image](https://user-images.githubusercontent.com/122420867/211709751-0357af9e-3f2c-4722-b96b-771ef5d5c086.png)
![image](https://user-images.githubusercontent.com/122420867/211709768-53ba343a-d005-471b-b004-733c20fa7eec.png)
![image](https://user-images.githubusercontent.com/122420867/211709780-6fb2b99e-eda5-4608-86b4-daf26b585642.png)
![image](https://user-images.githubusercontent.com/122420867/211709802-783a61b5-4ec1-41c2-846d-0524e9512c15.png)

### Export String

```true
!WA:2!1rvqVTrruym(qrRiIuRelL0dysBTaHsuIttoGO061SMKId2SEnPieYE2DND3PD9mdZmRtmCZcH4S)bWbFMt(qpJ6VGrrWFGCLB9xaVzTBBekzoy)MDEZB)EVVVVTqZsdlfwk833wWYuPekw0UNxRJ)gh)ZqXSJ)YP1KX(7(ThfVT482rrsSQWfYeui7SVF525iAqct0HrOkHx7oTCA6XVxirYtrJ9WNR6hXedrQ(7TZqc1frJXlFInLrX8kxtQ8(kYWxNwb(QbzsfBOjJE8qKc7IhHPQAzI03vGcuegvEORuHeklBcLOS8JG)KjwtucsCmwi)JQILHVWwnMJTZG0UuM5NxPUzrrKZN3Vr9UE976v31Zvya6TNKjX9nPoqx2hQFmwnrYXPPhhkTMAompVe9kZZ7SknsWbp)n1TJad11TBhNwTCPOHyP1SqSp828amioYPvNM9AnnJUeBwv1R9B6T03vVUU8fMYNWKkskrno5sZwKsHcEoYpfpy6BosSicR3CGEf(A9a4c4azMmybbokqQlRVJ(EwZHz9ZYOWeBeUgknLxA5KDXRVflMeW)WO8my0pw9jwvGLaRYe0kQFO2pAHPH8BJYlWYlDcle)NVd)oxlnUiW1WMHVceiybfL(DaFaL)xMIOe4ui8q9QnHw26LyKe3vjW0yvYTUimtKFCFdJjK4agnuoXKJz8PTShIi08lQ3)w6hOpqFiew6))e7igvnZ5CUalLNHglgTaafNb8Klo2iF2SSE9lGT(W4ng8c0q7ugkmkNJdsrszcW)PaNw2DywQIu(dgydkHGY6hB5MNqvFjHgNIfNw3Tv7gFT(XLTpP(x5miNihHtibP4EKeBj5NXMRDtUe05x1LOT8LSmra2NmKZeQL2VNU0(XxDH)RPaKx5)8sN0r9A23lxTotK3FM5LTcEntL40OCV6m72EETpXyxNCgteEQaXNC6YGzGSfhEkjuL8RV)LOmqJamrq(dSRdBNFfOZ34(V1Bxz7k3)T9a)U3S7gOJasopmXqqDHPYhvlJeoh5SNu2XL2wkepd0NKOX2gy6lXdbQZDVD2DNDNPybl5X3B0FTfbKLl6yaalaY(76gWszINuewZJmJgqAHuOIV6kJmZKzExNwon8A6w)eh(NEZioCm4Hjb9vjGskHLgo2hKKrKyRlGreuX288VfzjgVGDAmFbz1ihhfkuO4mJgMKN1Nv1nW81IM61kUfJV1N)WjJqcIXCR3WDekndx6arqIPzKBwvZEsXJEKUJzTYrp6FwhwR9VfNWfmWCOgRdQQh(6A9fput1BOzBv7anp)UfkwOO(NU(K2F3LjvSWIKMtOl6CaNLNdHbyhZ3ZKdkjDpyN935bLg93p9)(
```

**Comment by ne7ex** [comment]
https://clips.twitch.tv/DrabAmericanWaspBudBlast-u96-tiNE_F1K_07W

Heres how to replicate it, i tested it while respeccing ^

**Comment by InfusOnWoW** [comment]
The range checker is provided by a library found at https://github.com/WeakAuras/LibRangeCheck-2.0/

Though we can try to pinpoint which checker is misbehaving by entering into a chat window:
/run LibStub("LibRangeCheck-2.0"):checkAllCheckers()

That should show for each range which items are checked, the numbers for 33 and 35 would be interesting.

**Comment by ne7ex** [comment]
> The range checker is provided by a library found at https://github.com/WeakAuras/LibRangeCheck-2.0/
> 
> Though we can try to pinpoint which checker is misbehaving by entering into a chat window: /run LibStub("LibRangeCheck-2.0"):checkAllCheckers()
> 
> That should show for each range which items are checked, the numbers for 33 and 35 would be interesting.


Is there something I should do?

**Comment by InfusOnWoW** [comment]
Though we can try to pinpoint which checker is misbehaving by entering into a chat window: /run LibStub("LibRangeCheck-2.0"):checkAllCheckers()

**Comment by InfusOnWoW** [comment]
No response. In any case if you want to have it fixed, try reporting it on LibRangeCheck-2.0 repo. But do note, that likely no one has access to a max level mage on wotlk and thus your cooperation and willigness to run experiments is required.
